src/test/java/com/wikia/webdriver/testcases/desktop/navigation/localnav/CommunityHeaderTests.java

```diff
@@ -17,38 +17,38 @@
 
   @Test(groups = {"CommunityHeaderTests"})
   public void wordmarkShouldLinkToMainPage() {
-    MainPage mainPage = new CommunityHeader().clickWordmark();
+    MainPage mainPage = new CommunityHeaderDesktop().clickWordmark();
 
     Assertion.assertTrue(mainPage.isMainPage());
   }
 
   @Test(groups = {"CommunityHeaderTests"})
   public void wikiNameShouldLinkToMainPage() {
-    MainPage mainPage = new CommunityHeader().clickWikiName();
+    MainPage mainPage = new CommunityHeaderDesktop().clickWikiName();
 
     Assertion.assertTrue(mainPage.isMainPage());
   }
 
   @Test(groups = {"CommunityHeaderTests"})
   @Execute(onWikia = MobileWikis.DE_WIKI, language = "de")
   public void wikiNameOnNonEnglishWikiShouldLinkToMainPage() {
-    MainPage mainPage = new CommunityHeader().clickWikiName();
+    MainPage mainPage = new CommunityHeaderDesktop().clickWikiName();
 
     Assertion.assertTrue(mainPage.isMainPage());
   }
 
   @Test(groups = {"CommunityHeaderTests"})
   @Execute(asUser = User.ANONYMOUS)
   public void testAnonWikiButtons() {
-    CreateArticleModalComponentObject modal = new CommunityHeader().clickAddNewPage();
+    CreateArticleModalComponentObject modal = new CommunityHeaderDesktop().clickAddNewPage();
 
     Assertion.assertTrue(modal.isCreateNewArticleModalVisible());
   }
 
   @Test(groups = {"CommunityHeaderTests"})
   @Execute(asUser = User.USER)
   public void testLoggedInWikiButtons() {
-    CommunityHeader communityHeader = new CommunityHeader();
+    CommunityHeaderDesktop communityHeader = new CommunityHeaderDesktop();
 
     communityHeader.clickWikiActivity();
     Assertion.assertTrue(driver.getCurrentUrl().contains("Special:WikiActivity"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Special:WikiActivity" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)